config/locales/en.txt

```diff
@@ -1629,6 +1637,7 @@
   rss_log_title: Activity Log
   rss_by: by [user]
   rss_title: "[:app_title]"
+  rss_filtered_mouseover: "[:app_title] has applied filter(s), including any filter(s) in: (a) your Preferences and (b) Advanced Search or the search bar."
```

Comment:
Your help suggestion is perfect & easy to implement; will do.  
(In search bar, `images:` and `specimen:` apply filters.)